Fix syntax error if channel's name has quote

oliviernes opened the pull request:
Channels with quotes (example: L'Equipe) throw a syntax error when server read channels.conf . Quotes need also to be removed when reading m3u8 file to play the channel

oliviernes committed:
Fix syntax error if channel's name has quote

Channels with quotes (example: L'Equipe) throw a syntax error when server read channels.conf . Quotes need also to be removed when reading m3u8 file to play the channel

phpbg commented on src/Pages/Channels/M3u8.php:
```diff
@@ -72,7 +72,7 @@ public function __invoke(ServerRequestInterface $request)
 
         foreach ($channels as $channelDescriptor) {
-            $content .= "#EXTINF:-1 tvg-id=\"{$channelDescriptor['SERVICE_ID']}\",{$channelDescriptor['NAME']}\r\n";
+            $content .= "#EXTINF:-1 tvg-id=\"{$channelDescriptor['SERVICE_ID']}\",{$channelDescriptor['NAME']->replace("'","")}\r\n";
```
`'` should be supported. AFAIK m3u8 are utf-8 encoded.

See https://fr.wikipedia.org/wiki/M3U (french) with `'`examples...

Can you tell which m3u8 software do you use?

oliviernes replied:
Yes, I agree. It is unnecessary to remove quotes in the m3u8 file generator.

oliviernes committed:
Cancel removing quotes in M3u8.php

Quotes are supported in m3u8 file so there is no need to remove them as m3u8 are utf-8 encoded
